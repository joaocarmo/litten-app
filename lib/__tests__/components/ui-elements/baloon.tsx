import TestRenderer from 'react-test-renderer'
import ThemeProvider from '@components/theme/provider'
import { UIBalloon } from '@ui-elements'

describe('Snapshot test for the "UIBalloon" component', () => {
  it('renders correctly', () => {
    const element = TestRenderer.create(
      <ThemeProvider>
        <UIBalloon type="info">Some text here</UIBalloon>
      </ThemeProvider>,
    ).toJSON()
    expect(element).toMatchSnapshot()
  })
})
