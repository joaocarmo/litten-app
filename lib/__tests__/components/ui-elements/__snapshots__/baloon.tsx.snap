// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for the "UIBalloon" component renders correctly 1`] = `
<View
  style={
    [
      {
        "borderRadius": 12,
        "borderWidth": 0.5,
        "padding": 12,
      },
      {
        "backgroundColor": "#e0effb",
        "borderColor": "#84bff1",
      },
      undefined,
    ]
  }
>
  <Text
    style={
      [
        [
          {
            "color": "#000000",
            "fontSize": 14,
            "fontWeight": "200",
          },
          [
            {
              "fontWeight": "400",
            },
            {
              "color": "#6a6767",
            },
          ],
        ],
        {
          "marginBottom": 8,
          "marginTop": 8,
          "paddingBottom": 6,
          "paddingTop": 6,
        },
        undefined,
        undefined,
        undefined,
      ]
    }
  >
    Some text here
  </Text>
</View>
`;
